webapp/src/main/java/com/box/l10n/mojito/rest/View.java

```diff
@@ -7,6 +7,8 @@
  * @author jyi
  */
 public class View {
+  public interface Default {}
```

Comment:
Can you explain why this is needed? I expect it's related to @JSONVIEW but I'm not clear on the problem